test/integration/actions/action-create-session.spec.ts

```diff
@@ -177,4 +178,50 @@ describe('action-create-session', () => {
 			expect(result.slug).toMatch(/^session-/);
 		}
 	});
+
+	test('should create and store a hash of the token when creating a new session', async () => {
+		const plaintext = 'foobarbaz';
+		const hash = await bcrypt.hash(plaintext, BCRYPT_SALT_ROUNDS);
+		const user = await ctx.createUser(ctx.generateRandomWords(1), hash);
+
+		expect.assertions(1);
+		const result: any = await handler(
+			ctx.session,
+			actionContext,
+			user.contract,
+			makeRequest(ctx, {
+				password: plaintext,
+			}),
+		);
+
+		const sessionCard = await ctx.jellyfish.getCardById<SessionContract>(
+			ctx.context,
+			ctx.session,
+			result.id,
+		);
+
+		if (!isNull(sessionCard) && !isArray(sessionCard)) {
+			expect(typeof sessionCard?.data.token!.authentication).toBe('string');
```

Comment:
Same as my other comment below. We might want to check that the authentication token is a string that isn't empty.